concert/tests/integration/scenarios/test_experiment.py

```diff
@@ -0,0 +1,142 @@
+"""
+test_experiment.py
+------------------
+Encapsulates remote experiment test cases
+"""
+import asyncio
+import os
+import json
+from pathlib import Path
+from typing import List, Any, Dict
+import shutil
+from numpy import ndarray as ArrayLike
+import skimage.io as skio
+import zmq
+import concert
+from concert.experiments.addons import tango as tango_addons
+from concert.quantities import q
+from concert.devices.motors.dummy import LinearMotor, ContinuousRotationMotor
+from concert.devices.shutters.dummy import Shutter
+from concert.storage import RemoteDirectoryWalker
+from concert.networking.base import get_tango_device
+from concert.experiments.synchrotron import RemoteContinuousTomography
+from concert.devices.cameras.uca import RemoteNetCamera
+from concert.helpers import CommData
+from concert.tests import TestCase
+####################################################################################################
+# Docker daemon creates a DNS entries inside the specified network with the service names. We need to 
+# specify these domain names to communicate with a services on a given exposed port. In the compose.yml
+# we have specified `uca_camera` and `remote_walker` as service names for the mock camera and walker
+# tango server processes running inside their respective containers. Hence, in the session we'd
+# have to use these domain names. Moreover, we will use the environment variables, which were injected
+# to conainer process. These environment variables encapsulate relevant metadata from container
+# orchestration, which are necessary to write the test cases.
+####################################################################################################
+
+class TestRemoteExperiment(TestCase):
+
+    @staticmethod
+    def list_files(startpath: str) -> None:
+        for root, dirs, files in os.walk(startpath):
+            level: str = root.replace(startpath, '').count(os.sep)
+            indent: str = ' ' * 4 * (level)
+            print('{}{}/'.format(indent, os.path.basename(root)))
+            subindent: str = ' ' * 4 * (level + 1)
+            for f in files:
+                print('{}{}'.format(subindent, f))
+
+    async def asyncSetUp(self) -> None:
+        await super().asyncSetUp()
+        walker_host = os.environ["REMOTE_WALKER_HOST"]
+        walker_port = os.environ["REMOTE_WALKER_PORT"]
+        walker_dn = os.environ["REMOTE_WALKER_DN"]
+        self._walker_dev_uri = f"{walker_host}:{walker_port}/{walker_dn}#dbase=no"
+        self._num_darks = 10
+        self._num_flats = 10
+        self._num_radios = 100
+        self._servers = { "walker": CommData(os.environ["UCA_NET_HOST"],
+                                             port=os.environ["UCA_WALKER_ZMQ_PORT"],
+                                             socket_type=zmq.PUSH)}
+        self._camera = await RemoteNetCamera()
```

Comment:
For now, we can leave everything here as it is. We can think in the future if we want to make as system call "concert start test_experiment" here, but the whole PR only improves the tests, and we do not have to overdo it.